### PR TITLE
add endpoint for rustadventure subscribers

### PR DESCRIPTION
**Pull request by joelhooks**

### CODE REVIEW

File: apps/rustadventure/package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "rust-adventure",
+  "name": "rustadventure-turbo",
```
Comment (joelhooks) [inline apps/rustadventure/package.json:2]:
```suggestion
  "name": "rustadventure",
```

File: apps/rustadventure/src/components/forms/convertkit/index.tsx [hunk 2]
```diff
@@ -1,15 +1,38 @@
...
+    },
+    validationSchema: Yup.object({
+      email: Yup.string().required('Enter Your Email'),
+      first_name: Yup.string(),
+    }),
+
+    onSubmit: async (values) => {
+      // await new Promise((r) => setTimeout(r, 500))
+      setSubmitting(true)
+      axios.post('/api/convertkit/subscribe', values).then(({data}) => {
```
Comment (joelhooks) [inline apps/rustadventure/src/components/forms/convertkit/index.tsx:28]: this directly submits to our serverless API endpoint

File: apps/rustadventure/src/pages/index.tsx [hunk 3]
```diff
@@ -39,7 +41,9 @@ export default function Home() {
               </h3>
             </div>
             <div className="max-w-sm mx-auto">
-              <ConvertkitSubscribeForm />
+              <ConvertkitSubscribeForm
+                onSubmit={() => router.push('/confirmed')}
```
Comment (joelhooks) [inline apps/rustadventure/src/pages/index.tsx:45]: the redirect to `confirmed` happens here. We no longer hit the `confirm` endpoint, though we COULD

**Review by tayiorbeii — APPROVED** [review]
Looks good to me-- should we revisit later with a hand rolled double opt-in?

**Pull request merged**